### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'services'

### ISSUE THREAD

**Issue by vitor-caetano** [oracle/oci-cli#280]
All commands failing.

```
$ oci --version
2.10.1
$ oci os ns get
FileNotFoundError: [Errno 2] No such file or directory: 'services'
```

**Comment by HamadaGabrIbrahim** [comment]
@vitor-caetano Thanks for reporting this issue.
We are working on fixing this bug.

**Comment by brusMX** [comment]
Same here. 
This error message is really misleading because it has to do with the `~/.oci/config` file.
I have 2 profiles like this:

```conf
[DEFAULT]
user = ocid1.user.oc1..VALID_USER_1
fingerprint = valid_finger_print
key_file =key_generated_by_oci_setup_command
tenancy = ocid1.tenancy.oc1..VALID_TENANCY_1
region = valid_region_1

[profile1]
fingerprint=valid_finger_print_related_to_OCI_AUTH
key_file=LINK_TO_AUTOGEN_PEM_KEY_OCI_AUTH
tenancy=ocid1.tenancy.oc1..VALID_TENANCY_1
region=valid_region_1
security_token_file=TOKEN_GENERATED_BY_OCI_AUTHENTICATE
```

Now, this command returns my compartments correctly:

```bash
oci iam compartment list  --config-file ~/oci/config --profile profile1 --auth security_token

{[returns nice fancy json]}
```
But this profile just gives that error:
```
oci iam compartment list  --config-file ~/.oci/config --profile DEFAULT
FileNotFoundError: [Errno 2] No such file or directory: 'services'
```
So basically, this error message has nothing to do with the actual error. I believe the actual error has to do with the profile not having a valid security token ` --auth security_token`.

**Comment by Fustzecutor** [comment]
Same error here...I'm using certificates to authenticate.

Any update on fixing this please ? We cannot use at all the OCI buckets.

**Comment by vish1** [comment]
We have fixed this error in the latest release 2.10.4.

**Comment by AmitK0105** [comment]
FileNotFoundError: [Errno 2] No such file or directory: 'Data_OLAP_Sep_P1' Getting this error.. though the physical file is available in the directory.. Pls suggest
